src/httpd_jsonapi.c

```diff
@@ -458,6 +476,54 @@ fetch_playlist(const char *playlist_id)
   return playlist;
 }
 
+static int
+fetch_genres(struct query_params *query_params, json_object *items, int *total)
+{
+  json_object *item;
+  int ret = 0;
+  char*  bi;
+  char*  si;
```

Comment:
why two spaces before bi and si?
naming of bi and si could be better
assignment of ret does nothing